src/main/webapp/app/exercises/quiz/manage/re-evaluate/quiz-re-evaluate.component.scss

```diff
@@ -25,23 +28,36 @@
             display: inline-block;
         }
 
-        .input-group > input {
-            height: 100%;
-        }
-
         .form-group {
             display: flex;
-            align-items: center;
-            margin: 0 5px 15px;
+            flex-direction: column;
+            margin-bottom: 1.5rem;
+
+            .start-time {
+                width: 80%;
+            }
 
-            > * {
-                margin: 0 4px;
+            label {
+                font-size: 17px;
```

Comment:
There are css-variables for font-size as well

Reply:
Changed this to medium